notebooks/128-openvino-tokenizers/128-opnevino-tokenizers.ipynb

```diff
@@ -0,0 +1,507 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    test_strings = ["Test", "strings"]
</pre><p>maybe it is also good to compare result with hf tokenizer?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1697/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>